index.html

```diff
@@ -212,72 +224,76 @@
         .sizeByClinicalAttribute("Phos", "Result")
         .orderAllTooltipTables(["Source","Location"])
         .enableTrackTooltips(true)
-        .enableZoom(enableAdvanced) //zoom functionality avaiable only in the advance view
-        .advancedView(enableAdvanced)
-        .enableTrimmedTimeline(true)
-        .enableVerticalLine(false)
-        .enableHelperLines(true)
+        .plugins([
+          {
+            obj: new trimClinicalTimeline("Trim Timeline"), 
```

Comment:
Don't really understand why one would have to pass a name? This is only used by configCheckManager right?